scripts/test_env.py

```diff
@@ -36,20 +37,32 @@ def __init__(self, args: argparse.Namespace):
                           Please provide them in the `{args.root_dir}/configs/peer` directory")
             sys.exit(1)
         copy_or_prompt_build_bin("iroha", args.root_dir, peers_dir)
-
+        
         self.peers = [_Peer(args, i) for i in range(args.n_peers)]
 
         os.environ["IROHA2_CONFIG_PATH"] = str(peers_dir.joinpath("config.json"))
         os.environ["IROHA2_GENESIS_PATH"] = str(peers_dir.joinpath("genesis.json"))
         os.environ["IROHA_GENESIS_ACCOUNT_PUBLIC_KEY"] = self.peers[0].public_key
         os.environ["IROHA_GENESIS_ACCOUNT_PRIVATE_KEY"] = self.peers[0].private_key
+        
+        saved = self.peers[0].public_key
+        if is_startup_benchmark:
+            self.peers[0].public_key = "ed012072AED89EDB3CAA5EA203C6CED29A23CB387C9F5C48DB025E524DC508E18A8E22"
+            self.peers[1].public_key = "ed01200B06FCBBD0724DC11D25FC1A88B110B756712462DD6C0D762411E1A98B295AC8"
+            self.peers[2].public_key = "ed01203D922F3E7834F7AF20DAE2DE1AB2F48B4CF8FE95C62D372897692A64143F82C7"
+            self.peers[3].public_key = "ed01208BC55A04A1FC36A15DC63C59E8979FB7080E0937DDE11F2C3F783E9E167ABF4E"
 
         logging.info("Generating trusted peers...")
         self.trusted_peers = []
         for peer in self.peers:
             peer_entry = {"address": f"{peer.host_ip}:{peer.p2p_port}", "public_key": peer.public_key}
             self.trusted_peers.append(json.dumps(peer_entry))
         os.environ["SUMERAGI_TRUSTED_PEERS"] = f"[{','.join(self.trusted_peers)}]"
+        
+        self.peers[0].public_key = saved
```

Comment:
Why to save it, then change, then reload saved value?
Do some code between this assignments relies on that? If so then it's too implicit, it should be function parameters or something